src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/DateFormatTest.java

```diff
@@ -105,8 +107,10 @@ public void and() {
     public void or() {
         assertThat(
             query(SELECT_FROM +
-                  "WHERE date_format(insert_time, 'YYYY-MM-dd', 'UTC') < '2014-08-18' " +
-                  "OR date_format(insert_time, 'YYYY-MM-dd', 'UTC') > '2014-08-23'"),
+                  "WHERE date_format(insert_time, 'yyyy-MM-dd', 'UTC') < '2014-08-18' " +
+                  "OR date_format(insert_time, 'yyyy-MM-dd', 'UTC') > '2014-08-23'" +
+                  "ORDER BY insert_time " +
```

Comment:
Why are you modifying the query (order by) ?

Reply:
+1. There is a "containsInAnyOrder" below, not sure why "order by" here.

Reply:
not necessory, "**ORDER BY insert_time**" is copied from other integration test. will remove it.